app/livechat/client/views/app/livechatCurrentChats.js

```diff
@@ -31,6 +31,9 @@ Template.livechatCurrentChats.helpers({
 		return this.servedBy && this.servedBy.username;
 	},
 	status() {
+		if (localStorage.getItem('status')) {
```

Comment:
Why are you returning the localstorage value here? It's a helper to translate the status of each room rendered